egs/mini_an4/asr1/conf/train_conformer_transducer.yaml

```diff
@@ -0,0 +1,42 @@
+# minibatch related
+batch-size: 2
+maxlen-in: 800
+maxlen-out: 150
+
+# optimization related
+criterion: loss
+early-stop-criterion: "validation/main/loss"
+sortagrad: 0
+opt: noam
+epochs: 1
+report-interval-iters: 1
+accum-grad: 1
+patience: 0
+transformer-lr: 5.0
+transformer-warmup-steps: 10
+
+# network architecture
+## encoder related
+etype: transformer
+enc-block-arch:
+        - type: transformer
+          d_hidden: 4
+          d_ff: 4
+          heads: 2
+enc-block-repeat: 2
+## decoder related
+dtype: transformer
+dec-block-arch:
+        - type: transformer
+          d_hidden: 4
+          d_ff: 4
+          heads: 2
+## joint network related
+joint-dim: 4
+
+# transducer related
+## mtlalpha should be set to 1.0 (CTC) to use transducer
```

Comment:
Is there a way to force `mtlalpha: 1.0` internally in the transducer mode or whatever, and remove this config?
This will help to avoid some confusions.

Reply:
Thanks, forgot to change that part, there was only an assert to remove. Because of some recent changes, we don't need to force `mtlalpha` to 1 for transducer anymore.